src/main/java/CommandSet/AddCommand.java

```diff
@@ -0,0 +1,104 @@
+package CommandSet;
+
+import Helper.Storage;
+import Helper.TaskList;
+import Helper.Ui;
+
+import Tasks.Task;
+import Tasks.Todo;
+import Tasks.Deadline;
+import Tasks.Event;
+
+import Exceptions.MissingTimeArgumentException;
+
+/**
+ * <h1>AddCommand<h1/>
+ * <p>
```

Comment:
I like how you used HTML tags for JavaDocs

Reply:
Thanks for noticing! 😃